example/glue/configure_includes/configure_linux_riscv.mk

```diff
@@ -21,14 +21,17 @@
 
 include $(CONFIG_INCL_DIR)/configure_common.mk
 
-RISCV_CROSSTOOLS_PREFIX=riscv64-unknown-linux-gnu
-
 ifneq (,$(findstring _riscv64, $(SPEC)))
-	RISCV_TARGET=riscv64-unknown-linux-gnu
+	OMR_RISCV_CROSSTOOLS_PREFIX=$(RISCV_CROSSTOOLS_PREFIX)
 else
-	RISCV_TARGET=riscv32-unknown-linux-gnu
+	OMR_RISCV_CROSSTOOLS_PREFIX=riscv32-unknown-linux-gnu
+	ifeq (install, $(RISCV_TOOLCHAIN_TYPE))
+		OMR_RISCV_CROSSTOOLS_PREFIX=riscv32-linux-gnu
+	endif
 endif
 
+RISCV_TARGET=$(OMR_RISCV_CROSSTOOLS_PREFIX)

```

Comment:
What's the value of this macro? Why not just use `OMR_RISCV_CROSSTOOLS_PREFIX` below?

Reply:
Agreed and updated the script here and below with `RISCV_TARGET ` as suggested above.